storage_service/locations/forms.py

```diff
@@ -163,9 +163,11 @@ class Meta:
 
 
 class LocationForm(forms.ModelForm):
+    default = forms.BooleanField(required=False)
```

Comment:
The `default` is not a new field in the `Location` model, it uses `administration.models.Settings` instead. This is an artificial field we've added to `LocationForm`.